c/pthread-lit/fk2012_true-unreach-call.c

```diff
@@ -2,6 +2,9 @@ extern void __VERIFIER_error(void) __attribute__ ((__noreturn__));
 void __VERIFIER_assert(int cond) { if (!(cond)) { ERROR: __VERIFIER_error(); } return; }
 extern void __VERIFIER_assume(int);
 extern int __VERIFIER_nondet_int(void);
+extern void __VERIFIER_atomic_begin(void);
+extern void __VERIFIER_atomic_end(void);
```

Comment:
Why it is placed together with changes in ldv- benchmarks?

Reply:
The PR was addressing undeclared functions in general, not ldv benchmarks.